Integrations/integration-ArcSightESM.yml

```diff
@@ -136,7 +136,8 @@ script:
           }
           newIncidents.push({
             name: data.Name || ('New event from arcsight at ' + Date.now()),
-            labels: labels
+            labels: labels,
+            rawJSON: JSON.stringify(data[i]),
```

Comment:
You should remove `\`. I tried just stringify and it was throwing exception
`incident.rawJSON = JSON.stringify(event).replace(/\\"/g, '"');`